package.json

```diff
@@ -21,7 +21,7 @@
     "coffee": "^3.3.0",
     "egg-bin": "^1.0.0",
     "egg-ci": "^1.6.0",
-    "eslint": "^3.19.0"
```

Comment:
test on eslint@4 now, the only impact is [indent](http://eslint.org/docs/user-guide/migrating-to-4.0.0#-the-indent-rule-is-more-strict), and it can be fixed with `--fix` label